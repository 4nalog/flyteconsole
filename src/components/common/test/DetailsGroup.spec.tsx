import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { DetailsGroup } from '../DetailsGroup';

describe('DetailsGroup', () => {
    it('matches the known snapshot', () => {
        const items = [
            { name: 'item1', content: 'a simple string' },
            { name: 'item2', content: <div>A React node</div> }
        ];
        expect(
            renderer.create(<DetailsGroup items={items} />).toJSON()
        ).toMatchSnapshot();
    });
});
