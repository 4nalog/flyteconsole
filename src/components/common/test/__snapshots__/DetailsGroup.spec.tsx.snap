// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsGroup matches the known snapshot 1`] = `
<section
  className="makeStyles-rowsContainer-26"
>
  <div
    className="makeStyles-rowContainer-27"
  >
    <div
      className="makeStyles-groupLabel-28 makeStyles-textMuted-22"
      style={
        Object {
          "minWidth": "112px",
          "width": "112px",
        }
      }
    >
      item1
    </div>
    <div>
      a simple string
    </div>
  </div>
  <div
    className="makeStyles-rowContainer-27"
  >
    <div
      className="makeStyles-groupLabel-28 makeStyles-textMuted-22"
      style={
        Object {
          "minWidth": "112px",
          "width": "112px",
        }
      }
    >
      item2
    </div>
    <div>
      <div>
        A React node
      </div>
    </div>
  </div>
</section>
`;
